lib/web_console/templates/console.js.erb

```diff
@@ -1,3 +1,29 @@
+function getCookie(name) {
```

Comment:
Can you make those function private in the `REPLConsole` prototype like all other that are only necessary for `REPLConsole.prototype.install`

Reply:
Done